httpd/plan.sh

```diff
@@ -12,18 +12,21 @@ pkg_build_deps=(core/patch core/make core/gcc)
 pkg_bin_dirs=(bin)
 pkg_lib_dirs=(lib)
 pkg_expose=(80 443)
-pkg_svc_run="bin/httpd -DFOREGROUND -f $pkg_svc_config_path/httpd.conf"
 pkg_svc_user="root"
+pkg_svc_group="root"
 
 do_build() {
-  ./configure --prefix="$pkg_prefix" \
-              --with-expat="$(pkg_path_for expat)" \
-              --with-iconv="$(pkg_path_for libiconv)" \
-              --with-pcre="$(pkg_path_for pcre)" \
-              --with-apr="$(pkg_path_for apr)" \
-              --with-apr-util="$(pkg_path_for apr-util)" \
-              --with-z="$(pkg_path_for zlib)" \
-              --enable-ssl --with-ssl="$(pkg_path_for openssl)" \
-              --enable-modules=most --enable-mods-shared=most
-  make
+    ./configure --prefix="$pkg_prefix" \
+                --with-expat="$(pkg_path_for expat)" \
+                --with-iconv="$(pkg_path_for libiconv)" \
+                --with-pcre="$(pkg_path_for pcre)" \
+                --with-apr="$(pkg_path_for apr)" \
+                --with-apr-util="$(pkg_path_for apr-util)" \
+                --with-z="$(pkg_path_for zlib)" \
+                --with-ssl="$(pkg_path_for openssl)" \
```

Comment:
I'm not sure how the old directives worked here, as I believe we need the origin designation when using `pkg_path_for`. We should be explicit, though, and use `core/expat`, `core/libiconv`, etc.